Tribler/community/market/database.py

```diff
@@ -243,6 +243,29 @@ def add_transaction(self, transaction):
         for payment in transaction.payments:
             self.add_payment(payment)
 
+    def insert_or_update_transaction(self, transaction):
+        """
+        Inserts or updates a specific transaction in the database, according to the timestamp
```

Comment:
Please end this sentence with a period, and add that older transaction are not updated 👍

Reply:
Fixed.